package.json

```diff
@@ -913,4 +913,4 @@
     "yargs": "^15.4.1",
     "zlib": "^1.0.5"
   }
-}
\ No newline at end of file
+}
```

Comment:
This hit me a few times already :-)
I wonder why upstream doesn't fix it, hm. Let's keep it.